RecoBTag/Configuration/python/RecoBTag_cff.py

```diff
@@ -11,6 +11,7 @@
 from RecoBTag.TensorFlow.pfDeepDoubleX_cff import *
 from RecoBTag.MXNet.pfDeepBoostedJet_cff import *
 from RecoVertex.AdaptiveVertexFinder.inclusiveVertexing_cff import *
+from RecoBTag.PixelCluster.pixelCluster_cff import *
```

Comment:
If you import here just directly "RecoBTag.PixelCluster.pixelClusterTagInfos_cfi", there is no need of this include of "RecoBTag.PixelCluster.pixelCluster_cff"